src/containers/SearchPage/IntegerRangeFilter/FieldSelectIntegerRange.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+
+import { bool, string, number } from 'prop-types';
+import classNames from 'classnames';
+import { Field } from 'react-final-form';
+
+import { RangeSlider } from '../../../components';
+
+import css from './IntegerRangeFilter.module.css';
+
+const RADIX = 10;
+
+//  Ensures that the given values object has both minimum and maximum values.
+//  If `values` does not include `minValue` or `maxValue`, defaults are applied.
+const resolveMinMaxValues = (values, defaultMax, defaultMin) => {
+  const { maxValue, minValue } = values || {};
+  return { maxValue: maxValue || defaultMax, minValue: minValue || defaultMin };
```

Comment:
This causes an issue when minValue or maxValue is 0.

```suggestion
  return { 
    maxValue: maxValue != null ? maxValue : defaultMax, 
    minValue: minValue != null ? minValue : defaultMin,
  };
```